boards/arm/stm32_min_dev/stm32_min_dev_defconfig

```diff
@@ -35,3 +35,6 @@ CONFIG_CLOCK_STM32_AHB_PRESCALER=1
 # APB1 clock must not exceed 36MHz limit
 CONFIG_CLOCK_STM32_APB1_PRESCALER=2
 CONFIG_CLOCK_STM32_APB2_PRESCALER=1
+
+#enable I2C
+CONFIG_I2C=y
```

Comment:
@erwango, do you want this enabled by default?

Reply:
It should be removed. It depends on the application if the I2C should be enabled.

Reply:
Agree with ydamigos, I removed this.